zk/stages/stage_sequence_execute.go

```diff
@@ -304,6 +302,9 @@ func SpawnSequencingStage(
 		if err = sdb.hermezDb.WriteBlockL1InfoTreeIndex(blockNumber, l1TreeUpdateIndex); err != nil {
 			return err
 		}
+		if err = sdb.hermezDb.WriteBlockL1InfoTreeIndexProgress(blockNumber, infoTreeIndexProgress); err != nil {
+			return err
+		}
```

Comment:
I think we only need to store this information when we know it has changed.

Reply:
[This comment](https://github.com/0xPolygonHermez/cdk-erigon/pull/918#discussion_r1704513814) applies to this one as well.

Reply:
Yup, same thing, if we can keep it uniform everywhere that will be great.